src/test/java/org/takes/http/BkBasicTest.java

```diff
@@ -86,6 +99,70 @@ public void handlesSocket() throws IOException {
     }
 
     /**
+     * BkBasic can handle HTTP persistent connections.
+     * @throws Exception If some problem inside
+     */
+    @Test
+    @SuppressWarnings("PMD.DoNotUseThreads")
+    public void handlesPersistentConnection() throws Exception {
+        final int port = 8080;
+        final String uri = String.format("http://localhost:%d", port);
+        // @checkstyle MagicNumberCheck (1 line)
```

Comment:
@dmzaytsev  i need this suppression because of quality rules. To be compliant to J2EE, a webapp should not use any thread .